### PR TITLE
[Organizations] Add new organization page improvements

### PR DESCRIPTION
**Pull request by scottbommarito**
https://github.com/NuGet/NuGetGallery/issues/5725

(work to fix the breadcrumb at the top will be done in a separate PR with other breadcrumb work)

image

### CODE REVIEW

File: src/NuGetGallery/Controllers/OrganizationsController.cs [hunk 1]
```diff
@@ -74,11 +72,9 @@ public ActionResult Add()
             }
             catch (EntityException e)
             {
-                errorMessage = e.Message;
+                TempData["AddOrganizationErrorMessage"] = e.Message;
```
Comment (skofman1) [inline src/NuGetGallery/Controllers/OrganizationsController.cs]: UT?

**Comment by scottbommarito** [comment]
@anangaur tool-tips can't have links inside them. I can make the tool-tip itself a link to gravatar.com however.

File: src/NuGetGallery/Controllers/OrganizationsController.cs (hunk 1 above)
Comment (scottbommarito) [reply src/NuGetGallery/Controllers/OrganizationsController.cs]: There was one that was broken by this change that I have now fixed.

**Comment by scottbommarito** [comment]
as per offline discussion with @anangaur, using tool-tip but without link

**Pull request merged**